Assets/Tests/PlayModeTests/MigrationTests.cs

```diff
@@ -0,0 +1,477 @@
+﻿using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using Plugins.CountlySDK.Models;
+using Plugins.CountlySDK;
+using Newtonsoft.Json;
+using System.Web;
+using System.Collections.Specialized;
+using Newtonsoft.Json.Linq;
+using Plugins.CountlySDK.Helpers;
+using UnityEditor.PackageManager.Requests;
+using System.Linq;
+using iBoxDB.LocalServer;
+using Plugins.CountlySDK.Persistance.Entities;
+using Plugins.CountlySDK.Persistance.Repositories;
+using Plugins.iBoxDB;
+
+namespace Tests
+{
+    public class MigrationTests
+    {
+        private readonly string _serverUrl = "https://xyz.com/";
+        private readonly string _appKey = "772c091355076ead703f987fee94490";
+
+        /// <summary>
+        /// It validates the request migration on empty request repo.
+        /// </summary>
+        [Test]
+        public void TestMigrationOnEmptyRequestRepo()
+        {
+            CountlyConfiguration configuration = new CountlyConfiguration {
+                ServerUrl = _serverUrl,
+                AppKey = _appKey,
+            };
+
+            Countly.Instance.Init(configuration);
+            Countly.Instance.RequestHelper._requestRepo.Clear();
+
+            Assert.AreEqual(2, Countly.Instance.StorageHelper.SchemaVersion);

```

Comment:
This check is not future proof and will break in the future when another migration is added.